src/lib/dnssd/minimal_mdns/responders/QueryResponder.cpp

```diff
@@ -56,6 +56,7 @@ QueryResponderSettings QueryResponderBase::AddResponder(RecordResponder * respon
     {
         return QueryResponderSettings();
     }
+    ChipLogDetail(Discovery, "Responding with %s", responder->GetQName());
```

Comment:
GetQName is a FullQName, i.e. an array of names. This is why it is often not logged.

https://github.com/project-chip/connectedhomeip/blob/master/src/lib/dnssd/minimal_mdns/Logging.cpp#L57

Contains a `class QNameString` that you could extract to a separate header and use for logging purposes (it converts qnames into '.'-separated elements and does memory safety and overflow checks.

Or move this logic to the logging module (but that is controlled by HIGH_VERBOSITY ... maybe that is the right thing to do)

Reply:
Done, thanks for the guidance!

Sorry for the messy pull requests, still getting the hang of git (I use it sparingly)